service/src/main/resources/static/thyme.html

```diff
@@ -0,0 +1,27 @@
+<!--
```

Comment:
What's that? AFAIK we do not serve static HTML right now, even if it's in the JAR.

Reply:
Done